tests/serverspec/tls.yml

```diff
@@ -0,0 +1,114 @@
+- hosts: localhost
+  roles:
+    - reallyenglish.x509-certificate
+    - ansible-role-nginx
+  vars:
+    # XXX set this to no because the configuration contains releative path.
```

Comment:
i expected https://github.com/reallyenglish/ansible-role-nginx/pull/1#discussion_r128155262 meant this case. do you plan to write another case to test validation?